source/synthDrivers/sapi4.py

```diff
@@ -123,6 +129,16 @@ def speak(self,speechSequence):
 			elif isinstance(item, CharacterModeCommand):
 				textList.append("\\RmS=1\\" if item.state else "\\RmS=0\\")
 				charMode=item.state
+			elif isinstance(item, PitchCommand):
+				offset = int(config.conf["speech"]['sapi4']["capPitchChange"])

```

Comment:
I was looking at this code and this really feels incorrect to me. It looks like the caps pitch change is applied to every pitch change, regardless of whether it is related to capital letters.

Reply:
OK, I see you've opened a PR to try and fix this, that's great.